src/map/zone.h

```diff
@@ -573,6 +575,12 @@ class CZone
     void   SetLocalVar(const char* var, uint32 val);
     void   ResetLocalVars();
 
+    auto CreateTreasurePool(uint32 poolOwnerId, TREASUREPOOLTYPE poolType) const -> CTreasurePool&;
+    auto GetTreasurePool(CCharEntity* PChar) const -> CTreasurePool&;
+    auto GetTreasurePools() const -> std::unordered_map<uint32, CTreasurePool>&;
+    void ReleaseTreasurePool(CTreasurePool& PTreasurePool) const;
+    auto ReassignTreasurePool(CCharEntity* PPrev, CCharEntity* PNew) -> bool;
+
```

Comment:
We don't have this written down anywhere, but we're trying to move towards the convention of member functions (even bound ones) being `lowerCamelCase` and free functions (including in namespaces, and bindings) can be `UpperCamelCase`